jetbrains-core/src/software/aws/toolkits/jetbrains/services/dynamo/Attributes.kt

```diff
@@ -0,0 +1,93 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.dynamo
+
+import software.amazon.awssdk.services.dynamodb.model.AttributeValue
+import java.util.Base64
+
+typealias SearchResults = List<Map<String, DynamoAttribute<*>>>
+
+sealed class DynamoAttribute<T>(val value: T) {
+    abstract val dataType: String
+
+    open fun stringRepresentation() = value.toString()
+}
+
+class StringAttribute(value: String) : DynamoAttribute<String>(value) {
+    override val dataType: String = "S"
+
+    override fun stringRepresentation(): String = "\"$value\""
```

Comment:
nit: maybe we want to do like
```kt
const val QUOTE = '"'
```
So stuff like this is more readable

Reply:
went with    

```
 override fun stringRepresentation(): String = '"' + value + '"'
```